interfaces/cython/cantera/test/test_kinetics.py

```diff
@@ -405,20 +405,30 @@ def test_modify_thermo(self):
                 # Rate should be the same if reaction does not involve OH
                 self.assertAlmostEqual(w2[i] / w1[i], 1.0)
     def test_pdep_err(self):
-        err_msg = ("Invalid rate coefficient for reaction 'CH2CHOO <=> CH3O + CO'"
-                  "\nat P = 32018.699999999983 , T = 500.0\n"
-                  "Invalid rate coefficient for reaction 'CH2CHOO <=> CH3O + CO'"
-                  "\nat P = 32018.699999999983 , T = 1000.0\n"
-                  "Invalid rate coefficient for reaction 'CH2CHOO <=> CH3O + CO'"
-                  "\nat P = 101324.99999999999 , T = 500.0\n"
-                  "Invalid rate coefficient for reaction 'CH2CHOO <=> CH3O + CO'"
-                  "\nat P = 101324.99999999999 , T = 1000.0\n\n\n\n\n"
-                  "Invalid rate coefficient for reaction 'CH2CHOO <=> CH3 + CO2'"
-                  "\nat P = 10132499.999999985 , T = 500.0")
+        err_msg1 = "Invalid rate coefficient for reaction 'CH2CHOO <=> CH3O + CO'"
+        err_msg2 = "at P = 32018.699999999983 , T = 500.0"
+        err_msg3 = "Invalid rate coefficient for reaction 'CH2CHOO <=> CH3O + CO'"
+        err_msg4 = "at P = 32018.699999999983 , T = 1000.0"
+        err_msg5 = "Invalid rate coefficient for reaction 'CH2CHOO <=> CH3O + CO'"
+        err_msg6 = "at P = 101324.99999999999 , T = 500.0"
+        err_msg7 = "Invalid rate coefficient for reaction 'CH2CHOO <=> CH3O + CO'"
+        err_msg8 = "at P = 101324.99999999999 , T = 1000.0"
+        err_msg9 = "Invalid rate coefficient for reaction 'CH2CHOO <=> CH3 + CO2'"
+        err_msg10 = "at P = 10132499.999999985 , T = 500.0"
```

Comment:
Rather than 10 variables, you can make the same tuple as before (although you have to remove the `\n` from the strings), and then loop for the asserts:

```python
err_msg_list = str(e).splitlines()
for msg in err_msg:
    self.assertIn(msg, err_msg_list)
```

Reply:
Thanks for the instruction, I just applied the change